### PR TITLE
Doc blocks: Update ArgTable Reset button to use IconButton

### PR DESCRIPTION
**Pull request by MichaelArestad**
## What I did

ArgTable currently has a unique button design. This change updates to use Storybook's IconButton component.

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Review by domyen — APPROVED** [review]

File: code/lib/blocks/src/components/ArgsTable/ArgsTable.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const TableWrapper = styled.table<{
       },
 
       'th:last-of-type, td:last-of-type': {
```
Comment (domyen) [inline code/lib/blocks/src/components/ArgsTable/ArgsTable.tsx:69]: This applies globally to the `td` which might come back to bite us. 

I think it'd be safer to apply negative margin to the `StyledIconButton` that way we're scoping the positioning to only the element in question

**Review by domyen — CHANGES_REQUESTED** [review]

File: code/lib/blocks/src/components/ArgsTable/ArgsTable.tsx (hunk 1 above)
Comment (MichaelArestad) [reply code/lib/blocks/src/components/ArgsTable/ArgsTable.tsx:69]: Good call. On it.

**Review by domyen — APPROVED** [review]

**Comment by domyen** [comment]
lgtm

**Review by shilman — APPROVED** [review]
Nice one 💯

**Pull request merged**